torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1311,6 +1311,20 @@ XLATensor XLATensor::expand(const XLATensor& input, std::vector<int64_t> size) {
       GetExpandDimensions(input_shape.get(), std::move(size))));
 }
 
+XLATensor XLATensor::expand(const XLATensor& input,
+                            std::vector<torch::lazy::NodePtr>& size_nodes,
+                            const std::vector<int64_t> upper_bounds,
+                            const std::vector<bool> dynamic_dims) {
+  std::vector<XlaValue> size_values;
+  for (auto& size_node : size_nodes) {
+    size_values.push_back(
+        XlaValue(size_node));  // TODO: what do we set index to?
```

Comment:
index should be 0. We need a helper function to convert NodePtr to Value. Hold on to that since @wonjoolee95 is working on getting rid of `XlaValue` and use `torch::Lazy::Value` for pytorch/xla.

Reply:
I added `0` for now.